package.json

```diff
@@ -118,6 +118,7 @@
 				"rules": {
 					"@typescript-eslint/no-misused-promises": "off",
 					"@typescript-eslint/promise-function-async": "off",
+					"@typescript-eslint/strict-boolean-expressions": "off",
```

Comment:
cc @sindresorhus 

Have a look at these errors and where they appear. It doesn't let me use `if (!element) {return}`. I don't think that converting that to `typeof element === 'undefined'` makes it any better. 😬

If anything, `!element` and `element` are easier to tell apart than `typeof element === 'undefined' and `typeof element !== 'undefined'

https://github.com/sindresorhus/refined-github/runs/290686882

Reply:
I came to the same conclusion after having used it in the wild. Fixed: https://github.com/xojs/eslint-config-xo-typescript/releases/tag/v0.20.0